CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+
+### How to Contribute?
+
+If you want to contribute, start working through the Unstructured codebase, navigate to the Github "issues" tab and start looking through interesting issues. If you are not sure of where to start, then start by trying one of the smaller/easier issues here i.e. issues with the "good first issue" label and then take a look at the issues with the "contributions welcome" label. These are issues that we believe are particularly well suited for outside contributions, often because we probably won't get to them right now. If you decide to start on an issue, leave a comment so that other people know that you're working on it. If you want to help out, but not alone, use the issue comment thread to coordinate.
+
+
+## Pull-Request Checklist
+
+The following is a list of tasks to be completed before submitting a pull request for final review.
+
+### Before creating PR:
+
+1. Follow coding best practices
+    1. [ ] Make sure all new classes/functions/methods have docstrings.
+    1. [ ] Make sure all new functions/methods have type hints (optional for tests).
+    1. [ ] Make sure all new functions/methods have associated tests.
+    1. [ ] Update `CHANGELOG.md` and `__version__.py` if the core code has changed
+<br/><br/>
+1. Ensure environment is consistent
+    1. [ ] Update dependencies in `.in` files if needed (pay special attention to whether the current PR depends on changes to internal repos that are not packaged - if so the commit needs to be bumped).
+    1. [ ] If dependencies have changed, recompile dependencies with `make pip-compile`.
+    1. [ ] Make sure local virtual environment matches what CI will see - reinstall internal/external dependencies as needed.
+<br/><br/>    
+1. Run tests and checks locally
+    1. [ ] Run tests locally with `make test`. Some repositories have supplemental tests with targets like `make test-integration` or `make test-sample-docs`. If applicable, run these as well. Try to make sure all tests are passing before submitting the PR, unless you are submitting in draft mode.
+    1. [ ] Run typing, linting, and formatting checks with `make check`. Some repositories have supplemental checks with targets like `make check-scripts` or `make check-notebooks`. If applicable, run these as well. Try to make sure all checks are passing before submitting the PR, unless you are submitting in draft mode.
+<br/><br/>    
+1. Ensure code is clean
+    1. [ ] Remove all debugging artifacts.
+    1. [ ] Remove commented out code.
+    1. [ ] Double check everything has been committed and pushed, recommended that local feature branch is clean.
+    
+### PR Guidelines:
+
+1. [ ] PR title should follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) standards.
+
+    - PRs to private repositories should reference the JIRA issue the PR seeks to address, e.g. "feat CORE-123: Add X feature".
```

Comment:
i don't think community workers could work on private repositories or see the internal JIRA links